helpers/apiError.js

```diff
@@ -15,8 +15,7 @@ ApiError.prototype = new Error();
 
 ApiError.prototype.toJson = function () {
 	return {
-		message: this.message,
-		code: this.code
+		message: this.message
```

Comment:
According to swagger docs, in case of an error, the response is the following format:
```
{
  "error": "No data returned"
}
```
So, toJson should be changed accordingly.

Reply:
That's the thing I wanted to adjust in the specification. More projects seem to have a `message` field. [RFC 7807](https://tools.ietf.org/html/rfc7807) suggests using more specific error descriptions, so if we decide to extend the errors with some extra fields, `message` will probably stay unchanged. What do you think @Isabello?

Reply:
I support this. We should slowly walking towards #674

Reply:
we can standardize it according to industry standards in a subsequent PR ifwe arent already.